CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+   proposal reach its current state.
+
+Please note that this extra information is not a requirement for
+submitting new content. Contributors are trusted to use their judgment
+to decide whether or not the proposal needs this information. Arm
+recommends that you add this information so that it is easier for new
+ideas to be discussed and possibly accepted, especially for changes of
+great impact.
+
+Instead of a pull request, you can also create an
+[issue](https://github.com/ARM-software/abi/issues), in which you describe your
+proposed change. However beware that opening an issue will have less chance to
+make it into the ABI, as someone will need to do the leg-work to make actual
+changes.
+
+If you want to make ABI changes that for some reason can't be discussed in
+public, you can send an email to arm.eabi@arm.com.
+
+## Manual checking of the PDF documents and Continuous Integration
+
+To check the outcome of your changes, run the `tools/rst2pdf/generate-pdfs.sh`
+script. To install the (python) prerequisites for the script, run the
+`tools/rst2pdf/install.sh` script.
+
+You can also check the rst syntax of the documents you changed with the
+`tools/common/check-rst-syntax.sh` script.
+
+These scripts have only been tested on Linux.
+
+## Continuous integration
+
+Whenever you create a pull request, these syntax checking and PDF generating
+scripts will be executed by `Github Actions`.
+
+You can download the PDFs in the `Checks` tab of any pull request (in the
+`Checks` tab click on the `CI` summary page above the highlighted `Build`
+sub-tab).
+
+The CI bot will also check the `rst` syntax of the documents. You can check the
+output in the `Checks` tab in the `check .rst document syntax` build log
+dropdown.
+
+# Pull request workflow
+
+1. Make pull request
+
+    First you will have to make the actual pull request, which contains the
+    proposed changes. If you are unfamiliar with Github pull requests, you can
+    read up on them
+    [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+
+    New development (bug-fixes, proposals, extensions, and so on) is committed
+    on the `main` development branch. Therefore, please submit your PR against
+    the branch `main`.
+
+    If your change is substansive, please add a short entry to the `Change
+    history` section of the changed documents that makes clear what you changed.
+
+    Design documents are placed in the `<root>/design-documents` directory.
+
+2. Review of pull request
+
+    Your pull request needs to be reviewed. Anyone can review, but at least one
+    of the reviewers needs to be someone from Arm. Good candidate to review your
+    change would be the various so-called document owners who look after the
+    quality of the individual documents. Not every document has an owner, but
+    the most used and important ones do. See the table below:
```

Comment:
Your pull request needs to be reviewed. Anyone can review, but at least one
    of the reviewers needs to be someone from Arm. Good candidate to review your
    change would be the various so-called document owners who look after the
    quality of the individual documents. Not every document has an owner, but
    the most used and important ones do. See the table below:
    
    Change to
    
    Your pull request must be peer-reviewed. Anyone can review, but at least one reviewer must be someone from Arm. A good candidate to review your change would be the various document owners, who look after the quality of the individual documents.